edk2-stable202605 build fails

Building edk2-stable202605 fails ([full log](https://github.com/CascadeOS/ovmf-prebuilt/actions/runs/27074051739/job/79908228709)):
```
 > [20/46] RUN . venv/bin/activate && stuart_setup -c ArmVirtPkg/PlatformCI/QemuBuild.py -a AARCH64 Target=RELEASE TOOL_CHAIN_TAG=GCC5 BLD_*_NETWORK_HTTP_ENABLE=1 BLD_*_NETWORK_TLS_ENABLE=1 && stuart_update -c ArmVirtPkg/PlatformCI/QemuBuild.py -a AARCH64 Target=RELEASE TOOL_CHAIN_TAG=GCC5 BLD_*_NETWORK_HTTP_ENABLE=1 BLD_*_NETWORK_TLS_ENABLE=1 && stuart_build -c ArmVirtPkg/PlatformCI/QemuBuild.py -a AARCH64 Target=RELEASE TOOL_CHAIN_TAG=GCC5 BLD_*_NETWORK_HTTP_ENABLE=1 BLD_*_NETWORK_TLS_ENABLE=1:
100.5 INFO - --------------Cmd Output Finished---------------
100.5 INFO - --------- Running Time (mm:ss): 00:00 ----------
100.5 INFO - ----------- Return Code: 0x00000001 ------------
100.5 INFO - ------------------------------------------------
100.5 ERROR - Compiler #4000 from : Not available
100.5 CRITICAL - Build failed
100.5 PROGRESS - End time: 2026-06-06 21:19:28.426690	 Total time Elapsed: 0:00:00
100.5 SECTION - Log file is located at: /edk2/Build/BUILDLOG_ArmVirtPkg.txt
100.5 SECTION - Summary
100.5 ERROR - Error
------
Dockerfile:65
--------------------
  63 |     ENV build_target=RELEASE
  64 |     ENV stuart_opts="-c ArmVirtPkg/PlatformCI/QemuBuild.py -a AARCH64 Target=${build_target} TOOL_CHAIN_TAG=${toolchain} ${common_build_opt}"
  65 | >>> RUN . venv/bin/activate && stuart_setup ${stuart_opts} && stuart_update ${stuart_opts} && stuart_build ${stuart_opts}
  66 |     
  67 |     # Build RiscV.
--------------------
ERROR: failed to build: failed to solve: process "/bin/sh -c . venv/bin/activate && stuart_setup ${stuart_opts} && stuart_update ${stuart_opts} && stuart_build ${stuart_opts}" did not complete successfully: exit code: 1
Error: command failed: ExitStatus(unix_wait_status(256))
Error: Process completed with exit code 1.
```

--- a/xtask/Dockerfile
+++ b/xtask/Dockerfile
@@ -49,7 +49,7 @@ RUN ln -s /edk2-platforms/Silicon /edk2/Silicon && \
 # Set up deps.
 RUN python3 -m venv venv
 RUN . venv/bin/activate && pip install -r pip-requirements.txt --upgrade
-ENV toolchain=GCC5
+ENV toolchain=GCC
 RUN . venv/bin/activate && python BaseTools/Edk2ToolsBuild.py -t "${toolchain}"
 
 RUN sed -i '/\[Components\]/a \  Silicon/Intel/IntelSiliconPkg/Feature/VTd/IntelVTdDxe/IntelVTdDxe.inf' \
@@ -64,13 +64,13 @@ ENV common_build_opt="BLD_*_NETWORK_HTTP_ENABLE=1 BLD_*_NETWORK_TLS_ENABLE=1"
 ENV tpm_build_opt="BLD_*_TPM1_ENABLE=1 BLD_*_TPM2_ENABLE=1"
 
 # Build AARCH64.
-ENV GCC5_AARCH64_PREFIX="/usr/bin/aarch64-linux-gnu-"
+ENV GCC_AARCH64_PREFIX="/usr/bin/aarch64-linux-gnu-"
 ENV build_target=RELEASE
 ENV stuart_opts="-c ArmVirtPkg/PlatformCI/QemuBuild.py -a AARCH64 Target=${build_target} TOOL_CHAIN_TAG=${toolchain} ${common_build_opt}"
 RUN . venv/bin/activate && stuart_setup ${stuart_opts} && stuart_update ${stuart_opts} && stuart_build ${stuart_opts}
 
 # Build RiscV.
-ENV GCC5_RISCV64_PREFIX="/opt/riscv/bin/riscv64-unknown-elf-"
+ENV GCC_RISCV64_PREFIX="/opt/riscv/bin/riscv64-unknown-elf-"
 ENV build_target=RELEASE
 ENV stuart_opts="-c OvmfPkg/PlatformCI/QemuBuild.py -a RISCV64 Target=${build_target} TOOL_CHAIN_TAG=${toolchain} ${common_build_opt}"
 RUN . venv/bin/activate && stuart_setup ${stuart_opts} && stuart_update ${stuart_opts} && stuart_build ${stuart_opts}
@@ -81,7 +81,7 @@ ENV stuart_opts="-c OvmfPkg/PlatformCI/PlatformBuild.py -a X64 Target=${build_ta
 RUN . venv/bin/activate && stuart_setup ${stuart_opts} && stuart_update ${stuart_opts} && stuart_build ${stuart_opts}
 
 # Build LoongArch64.
-ENV GCC5_LOONGARCH64_PREFIX="/opt/loongarch64_cross-toolchain/cross-tools/bin/loongarch64-unknown-linux-gnu-"
+ENV GCC_LOONGARCH64_PREFIX="/opt/loongarch64_cross-toolchain/cross-tools/bin/loongarch64-unknown-linux-gnu-"
 ENV build_target=RELEASE
 ENV stuart_opts="-c OvmfPkg/PlatformCI/QemuBuild.py -a LOONGARCH64 Target=${build_target} TOOL_CHAIN_TAG=${toolchain} ${common_build_opt}"
 RUN . venv/bin/activate && stuart_setup ${stuart_opts} && stuart_update ${stuart_opts} && stuart_build ${stuart_opts}

--- a/xtask/Dockerfile
+++ b/xtask/Dockerfile
@@ -10,9 +10,13 @@ RUN apt-get update && apt-get install -y \
     python3 \
     python3-dev \
     python3-venv \
+    rustup \
     uuid-dev \
     wget \
     xz-utils
+
+RUN rustup install 1.96
+RUN cargo install --locked cargo-make@0.37.24
 
 # Use the 2023.09.13 nightly GCC 5 toolchain for RISCV64.
 RUN wget -q -O gcc-riscv.tar.xz https://github.com/riscv-collab/riscv-gnu-toolchain/releases/download/2025.09.28/riscv64-elf-ubuntu-22.04-gcc-nightly-2025.09.28-nightly.tar.xz

--- a/xtask/Dockerfile
+++ b/xtask/Dockerfile
@@ -1,4 +1,4 @@
-FROM docker.io/debian:bookworm-slim
+FROM docker.io/debian:trixie-slim
 
 RUN apt-get update && apt-get install -y \
     g++ \
@@ -11,7 +11,8 @@ RUN apt-get update && apt-get install -y \
     python3-dev \
     python3-venv \
     uuid-dev \
-    wget
+    wget \
+    xz-utils
 
 # Use the 2023.09.13 nightly GCC 5 toolchain for RISCV64.
 RUN wget -q -O gcc-riscv.tar.xz https://github.com/riscv-collab/riscv-gnu-toolchain/releases/download/2025.09.28/riscv64-elf-ubuntu-22.04-gcc-nightly-2025.09.28-nightly.tar.xz

